src/mailmime_disposition.rs

```diff
@@ -82,7 +72,8 @@ pub unsafe fn mailmime_disposition_parse(
             }
             match current_block {
                 652864300344834934 => {
-                    dsp = mailmime_disposition_new(dsp_type, list);
+                    // FIXME: move list instead of cloning once error handling is rustified
+                    dsp = mailmime_disposition_new(dsp_type, list.clone());
```

Comment:
Have to clone here to be able to iterate over the list below.

Going to cleanup `mailmime_disposition_parm` and `unnamed_3` (that was probably a `union`) next so I can store `mailmime_disposition_parm` instead of a pointer to it.